### PR TITLE
Improve the customize button

### LINKED ISSUE #68: 🚀 Feature Request: Improve the customize button
### Problem Overview 👁️‍🗨️ 

It may not be obvious to users that a pencil icon is meant to customize the practice session 

### What would you like? 🧰

- [x] The button should have an icon that can be familiar to users, such as a cog
image

- [x] Add a tooltip that indicates that this button customizes the practice session when hovered.

### What alternatives have you considered? 🔍

NA

### Additional details ℹ️

NA

### PR DESCRIPTION
**Pull request by soumyankar**
Fixes #68 

- Changed icon to a cog
- Added tooltip on hover

### CODE REVIEW

File: packages/app/src/components/CustomizePracticeCard.vue [hunk 1]
```diff
@@ -7,8 +7,13 @@
         size="20px"
         round
         text-color="#C4C4C4"
```
Comment (ArsalaBangash) [inline packages/app/src/components/CustomizePracticeCard.vue:9]: Please change the text-color (icon color) to the one specified in the issue.

File: packages/app/src/components/CustomizePracticeCard.vue [hunk 2]
```diff
@@ -7,8 +7,13 @@
...
-        icon="mdi-pencil"
-      />
+        icon="settings"
+        tooltip="some text"
+      >
+      <q-tooltip anchor="center right" self="center left" :offset="[10, 10]" content-class="bg-primary" content-style="font-size: 16px">
```
Comment (ArsalaBangash) [inline packages/app/src/components/CustomizePracticeCard.vue:13]: I think the tooltip would be more natural to the bottom. What do you think?

File: packages/app/src/components/CustomizePracticeCard.vue [hunk 3]
```diff
@@ -7,8 +7,13 @@
...
+        Customize Practice Session
```
Comment (ArsalaBangash) [inline packages/app/src/components/CustomizePracticeCard.vue:14]: Maybe 'Customize Session' would be shorter and still descriptive enough. What do you think?

File: packages/app/src/pages/CustomizePracticePage.vue [hunk 4]
```diff
@@ -35,25 +35,38 @@
         </q-step>
 
         <template v-slot:navigation>
-          <q-stepper-navigation class="justify-center">
+          <q-stepper-navigation class="flex justify-center q-gutter-sm">
```
Comment (ArsalaBangash) [inline packages/app/src/pages/CustomizePracticePage.vue:38]: Is this file change supposed to be a part of this PR?

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
Solid iteration :)

File: packages/app/src/pages/CustomizePracticePage.vue (hunk 4 above)
Comment (soumyankar) [reply packages/app/src/pages/CustomizePracticePage.vue:38]: Oh no, it;s not supposed to be. I think I added that to my previous PR but forgot to push it!!

Comment (ArsalaBangash) [reply packages/app/src/pages/CustomizePracticePage.vue:38]: Let's revert the changes from this PR then :) 

You can do so via git checkout master  packages/app/src/pages/CustomizePracticePage.vue

File: packages/app/src/components/CustomizePracticeCard.vue [hunk 5]
```diff
@@ -186,7 +190,7 @@ export default {
 }
 #btn-customize-practice {
   z-index: 4;
-  color: gray !important;
+  color: white !important;
```
Comment (ArsalaBangash) [inline packages/app/src/components/CustomizePracticeCard.vue:193]: What I meant was the color of the icon should be #114489, and the background should remain white :)

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]
Solid iteration

**Review by ArsalaBangash — APPROVED** [review]
Excellent work @soumyankar

**Pull request merged**